.github/workflows/tests.yml

```
@@ -19,5 +19,12 @@ jobs:
       - uses: actions/checkout@ac593985615ec2ede58e132d2e21d2b1cbd6127c
       - name: Run tests
         run: make test-exercises
+  generator:
+    name: Check Generator Templates
+    runs-on: ubuntu-latest
```

I would recommend to pin to a specific version:

```suggestion
    runs-on: ubuntu-22.04
```

This helps prevent potential unforeseen breakages in the future. p.s. I know the other action in this workflow also uses `ubuntu-latest`, so feel free to merge this and update both in a follow-up PR.

https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources